.github/ISSUE_TEMPLATE/new_chapter.yml

```
@@ -0,0 +1,67 @@
+name: "\U0001F4D6 New Chapter Template 2"
+description: Template for proposing a new chapter
+title: "<title>"
+labels: book
+body:
+- type: markdown
+  attributes:
+    value: "Please complete the following sections to propose a new chapter. If you have the necessary permissions, please also assign the appropriate label (or labels) to your issue, otherwise at TTW team member will assign labels later."
+- type: input
+  id: title
+  attributes:
+    label: Title
+    description: |
+      Please provide a proposed title
+  validations:
+    required: true
+- type: input
+  id: guide
+  attributes:
+    label: Guide
+    description: |
+      Which Guide will you chapter be a part of? Please reference the [book](https://the-turing-way.netlify.app/welcome.html)'s table of contents to see which Guides make up the book
+  validations:
+    required: true
+- type: input
+  id: branch
+  attributes:
+    label: Branch
+    description: |
+      Please provide the name of the branch you are working from
+- type: textarea
+  id: summary
+  attributes:
+    label: Summary of proposed chapter
+    description: |
+      Please provide a detailed description of the chapter you are proposing. 
+      - What is the purpose of your chapter?
+      - Why are you proposing it?
+  validations:
+    required: true
+- type: textarea
+  id: resources
+  attributes:
+    label: Resources
+    description: List and describe any external resources that may be helpful when writing the chapter. We suggest using bullets (-) and/or checkboxes (-[])
```

```suggestion
    description: List and describe any external resources that may be helpful when writing the chapter. We suggest using bullets (-) and/or checkboxes (- [ ])
```

Just in case they try to directly copy and then find that it doesn't work